main.go

```diff
@@ -366,11 +295,11 @@ This means that when the API changes
 	}
 
 	envs := []string{
-		fmt.Sprintf("DELIVER_PASSWORD=%s", configs.Password),
+		fmt.Sprintf("DELIVER_PASSWORD=%s", string(configs.Password)),
 	}
 
-	if configs.AppPassword != "" {
-		envs = append(envs, fmt.Sprintf("FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD=%s", configs.AppPassword))
+	if string(configs.AppPassword) != "" {
+		envs = append(envs, fmt.Sprintf("FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD=%s", string(configs.AppPassword)))
```

Comment:
same as above

Reply:
Fixed in:
https://github.com/bitrise-io/steps-deploy-to-itunesconnect-deliver/pull/61/commits/228861e85a97e1f9399ec819aed8dc1e70e2ef4e